org.chocolate_doom.ChocolateDoom.yml

```diff
@@ -0,0 +1,30 @@
+app-id: org.chocolate_doom.ChocolateDoom
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: chocolate-doom
+rename-desktop-file: chocolate-doom.desktop
+rename-appdata-file: doom.appdata.xml
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --share=ipc
+  - --device=dri
+
+modules:
+  - name: chocolate-doom
+    buildsystem: autotools
+    build-options:
+      cflags: -fcommon
+    sources:
+      - type: git
+        url: https://github.com/chocolate-doom/chocolate-doom.git
+        tag: chocolate-doom-3.0.1
+        commit: 539ef367c9982c5945f8608b63d2236eaecc929a
+      - type: patch
+        path: appdata.patch
+    post-install:
+      - mkdir -p ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/
+      - mv ${FLATPAK_DEST}/share/icons/chocolate-doom.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png
```

Comment:
are you sure it doesn't expect the icon there otherwiser? I'd just use install here to copy it.

Reply:
In the .rpm world, you would try to avoid duplication like this, but I think you are right. Better safe than sorry.